docs/src/main/java/io/helidon/docs/mp/guides/TestingJunit5Snippets.java

```diff
@@ -92,6 +93,12 @@ void testConfiguredGreeting(WebTarget webTarget) {
                 validate(webTarget, "/greet", "Unite World!");
             }
 
+            @Test
+            @AddConfigBlock("app.greeting=Unite")
```

Comment:
The example should illustrate what we can do with this annotation that we cannot do with the other...declare _multiple_ settings as in the text you added to the doc.

Reply:
The problem is that the `helidon/examples/quickstarts/helidon-quickstart-mp` only uses one config property, so there is no point to add more.

I cannot create new config properties to inject in the test as a field, because of:
`java.lang.RuntimeException: When a class is annotated with @HelidonTest(resetPerTest=true), injection into fields or constructor is not supported, as each test method uses a different CDI container. Field private java.lang.String io.helidon.microprofile.tests.testing.junit5.TestAddConfigBlockProperties.value1 is annotated with @Inject`

Basically I need to create a new Snippet to test something like:
```
@HelidonTest
@AddConfigBlock("""
    some.key1=some.value1
    some.key2=some.value2
""")
class TestAddConfigBlockProperties {

    @Inject
    @ConfigProperty(name = "some.key1")
    private String value1;

    @Inject
    @ConfigProperty(name = "some.key2")
    private String value2;

    @Test
    void testValue() {
        assertThat(value1, is("some.value1"));
        assertThat(value2, is("some.value2"));
    }
}

```
Do you agree with it?

Reply:
I have pushed the change. Just let me know whether you are fine with it.